dcos_installer/cli.py

```diff
@@ -254,13 +259,22 @@ def get_argument_parser():
 
     mutual_exc.add_argument(
         '--set-superuser-password',
-        action=PasswordAction,
+        action=ArgsAction,
         metavar='password',
         dest='password',
         nargs='?',
         help='Hash the given password and store it as the superuser password in config.yaml'
     )
 
+    mutual_exc.add_argument(
+        '--generate-node-upgrade-script',
+        action=ArgsAction,
+        metavar='current_version',
```

Comment:
`current` could be vague to the user -- are we asking about the current version of the dgc or the current version that's running on the cluster? Maybe `installed_version` or `upgrade_from_version` would be clearer?